tests/tests_selinux_disabled.yml

```diff
@@ -81,3 +82,10 @@
         name: sar-user
         remove: yes
         state: absent
```

Comment:
I think those three tasks above should be also tagged as `[ 'tests::cleanup', 'tests::tier1::cleanup' ]`. Maybe use a block?

Reply:
Applies also to "Backup original /etc/selinux/config", one does not need to back it up, if one does not need to restore it.

Reply:
Fixed.